cumulus_library/studies/core/core_templates/condition.sql.jinja

```diff
@@ -61,7 +61,7 @@ WITH temp_condition AS (
 SELECT
     tc.id,
     cdc.code AS category_code,
-    cdc.code_system AS category_code_system,
+    cdc.code_system AS category_system,
```

Comment:
This approach is fine, but I worry that we're setting ourselves up for accidentally using `code_system` again down the line by not changing the denorm code now to give `.system` instead of `.code_system`.